StandaloneExample/StandaloneExample.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Terminal.Gui" Version="0.17.0" />
+    <ProjectReference Include="..\Terminal.Gui\Terminal.Gui.csproj" />
+    <!-- PackageReference Include="Terminal.Gui" Version="0.17.0" / -->

```

Comment:
This is intended to be a standalone example, one that does not require the current project.   If you want to just try things out with the released project, you can edit "Example" in the default solution.